src/loggers/WinstonErrorLoggyslav.ts

```diff
@@ -0,0 +1,10 @@
+import {ErrorLoggerInterface} from "../interfaces/ErrorLoggerInterface";
+import {ErrorObject, LogOptions} from "../interfaces/LoggerInterface";
+import {WinstonLoggyslav} from "./WinstonLoggyslav";
+
+export class WinstonErrorLoggyslav extends WinstonLoggyslav implements ErrorLoggerInterface {
+    public logError(errorObject: ErrorObject, options?: LogOptions): void {
```

Comment:
Do you really need parameter `options` here, since it's not used?

Reply:
Well, it is for future purposes, I have so much on my mind and this is so I don't forget 👍